sdcm/nemesis.py

```diff
@@ -1877,6 +1882,44 @@ def disrupt_grow_shrink_cluster(self):
 
         self.log.info("Finish cluster shrink. Current number of nodes %s", len(self.cluster.nodes))
 
+    def disrupt_hot_reloading_internode_certificate(self):
+        '''
+        https://github.com/scylladb/scylla/issues/6067
+        Scylla has the ability to hot reload SSL certificates.
+        This test will create and reload new certificates for the inter node communication.
+        '''
+        self._set_current_disruption('HotReloadInternodeCertificate')
+        if not self.cluster.params.get('server_encrypt', None):
```

Comment:
does scylla support client facing hot reloading ?

Reply:
IIUC it could, but this test is about inter node certification.

Reply:
@fgelcer open yourself a task to test also client encryption, it can be lower priority.